.github/workflows/validate-appstream.yml

```
@@ -19,11 +19,11 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: '3.11'
-    - name: Install appstream-util
+    - name: Install appstream
       run: |
         sudo apt-get update
         sudo apt-get install appstream-util gettext
```

```suggestion
        sudo apt-get install appstream gettext
```

Not sure appstream-util is still needed

---

I wanted to change that, but only changed it in the step's description. Oh well, will fix.